src/blockchain.js

```diff
@@ -194,6 +194,16 @@ class Blockchain {
 
     this.pendingTransactions.push(transaction);
   }
+  
+  /**
+   * Adds an initial amount to the target address
+   * @param {string} walletNumber 
```

Comment:
please add an trailing line, for code style (travis)

Reply:
I don't think that real blockchain implementations have a seeding function, so we shouldn't do it either. The only way to get coins is to work for them. 

Maybe we can move this function to the test suite? It should set the difficulty very low and mine a few blocks for the given wallet. I think that makes more sense. What's your opinion?

Reply:
Yeah, moving it to the test suite seems to be a better choice at this phase of the project.